tests/components/dynalite/test_cover.py

```diff
@@ -33,7 +35,7 @@ async def test_cover_setup(hass, mock_device):
         entity_state.attributes["current_tilt_position"]
         == mock_device.current_cover_tilt_position
     )
-    assert entity_state.attributes["device_class"] == mock_device.device_class
+    assert entity_state.attributes[ATTR_DEVICE_CLASS] == mock_device.device_class
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6212145)